mention_subgroup

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+
+import sys
+
+
+def mention_subgroup(groups, sender, groups_list):
+    mentions = ''
+
+    for group in groups:
+        if group[:2].upper() not in groups_list.keys():
```

Comment:
You should be able to simplify the way this reads by just using ``if groups in list: do stuff`` rather than ``if not in list: continue, else do stuff``